lib/ed25519.rb

```diff
@@ -28,6 +28,7 @@ class << self
 
   # Select the Ed25519::Provider to use based on the current environment
   if defined? JRUBY_VERSION
+    require "jruby"
```

Comment:
Added this line because JRuby 9.3.0 was throwing `undefined method 'runtime' for JRuby:Module` error.

Referred to https://github.com/jruby/jruby/issues/6125 for fix.